### ISSUE TITLE
Write a guide on how to use Frames for reading tabular data (stuff like CSV)

### ISSUE THREAD

**Issue by NickSeagull** [DataHaskell/docs#30]
[Frames](https://github.com/acowley/Frames)

**Comment by ocramz** [comment]
Frames does have a nice tutorial : https://acowley.github.io/Frames/

**Comment by NickSeagull** [comment]
Yes, but I mean something "for dummies" :smile: . Like, why the extensions, which things require what and more in detail
